src/cursor/cur_backup.c

```diff
@@ -676,9 +671,6 @@ __backup_config(WT_SESSION_IMPL *session, WT_CURSOR_BACKUP *cb, const char *cfg[
     /*
      * Compatibility checking.
      *
-     * Log remove cannot mix with log-file based incremental backups (but if a duplicate cursor,
-     * removal has been temporarily suspended).
-     *
      * Duplicate backup cursors are only for log targets or block-based incremental backups. But log
```

Comment:
Remove "log targets or"

Reply:
That is actually correct. MongoDB uses duplicate backup cursors with `target=(log:)` during incremental backup to catch-up/extend the backup cursor to get log files/records that occur during the backup process.

Reply:
Are those duplicate backup cursors for log-based incremental backup, block-bases incremental backup, or both?

Reply:
Weird - I'm sure I wrote a reply for this yesterday but it's vanished into the ether somehow. Anyway, all backup cursors (including log- and block-based incremental) can be extended. There's a bit more information in the documentation: https://source.wiredtiger.com/develop/backup.html#backup_duplicate